src/PointwiseFunctions/Hydro/Tags.hpp

```diff
@@ -191,6 +192,27 @@ struct SpecificInternalEnergy : db::SimpleTag {
   static std::string name() noexcept { return "SpecificInternalEnergy"; }
 };
 
+/// The vector \f$J^i\f$ in \f$\dot{M} = -\int J^i s_i d^2S\f$,
+/// representing the mass flux through a surface with normal \f$s_i\f$.
+/// Note that the integral is understood
+/// as a flat-space integral: all metric factors are included in \f$J^i\f$.
+/// In particular, if the integral is done over a Strahlkorper, the
+/// euclidean_area_element of the Strahlkorper should be used, and \f$s_i\f$ is
```

Comment:
please put `euclidean_area_element` in ticks and make sure it links correctly in Doxygen